doc/RELEASE_WALKTHROUGH.rst.txt

```diff
@@ -99,25 +101,16 @@ may have been accessed and changed by someone else and a push will fail::
     $ git branch <new version>  # only when starting new numpy version
     $ git checkout v1.14.x  # v1.14.x already existed for the 1.14.4 release
 
-Edit the ``.travis.yml`` and ``.appveyor.yml`` files to make sure they have the
-correct version, and put in the commit hash for the ``REL`` commit created
-above for ``BUILD_COMMIT``, see the _example from `v1.14.3`::
-
-    $ gvim .travis.yml .appveyor.yml
-    $ git commit -a
-    $ git push upstream HEAD
+Edit the ``azure/posix.yml`` and ``azure/windows.yml`` files to make sure they
+have the correct version, and put in the commit hash for the ``REL`` commit
+created above for ``BUILD_COMMIT``. Push and, if you are on a fork of the repo,
+merge the branch.
 
-Now wait. If you get nervous at the amount of time taken -- the builds can take
-several hours-- you can check the build progress by following the links
-provided at `<https://github.com/MacPython/numpy-wheels>`_ to check the travis
-and appveyor build status. Check if all the needed wheels have been built and
+Check if all the needed wheels have been built and
 uploaded before proceeding. There should currently be 22 of them at
-`<https://wheels.scipy.org>`_, 4 for Mac, 8 for Windows, and 10 for Linux.
-Note that sometimes builds, like tests, fail for unrelated reasons and you will
-need to restart them.
-
-.. example_: https://github.com/MacPython/numpy-wheels/commit/fed9c04629c155e7804282eb803d81097244598d
-
+`<https://anaconda.org/multibuild-wheels-staging/numpy>`_, 4 for Mac, 8 for
+Windows, and 10 for Linux.  Note that sometimes builds, like tests, fail for
+unrelated reasons and you will need to restart them.
```

Comment:
Again, I'd just replace the old names and links with the new ones.

Reply:
ok